packages/datetime/src/timePicker.tsx

```diff
@@ -257,21 +280,25 @@ export class TimePicker extends React.Component<TimePickerProps, ITimePickerStat
     }
 
     private renderInput(className: string, unit: TimeUnit, value: string) {
-        const isValid = isTimeUnitValid(unit, parseInt(value, 10));
+        const valueNumber = parseInt(value, 10);
+        const isValid = isTimeUnitValid(unit, valueNumber);
         const isHour = unit === TimeUnit.HOUR_12 || unit === TimeUnit.HOUR_24;
 
         return (
             <input
+                aria-valuenow={valueNumber}
```

Comment:
let's leave a comment about why we're doing this -- it's a little unintuitive since this is _almost_ the correct semantic element for this use case, which would obviate the need for this attribute. also I think we should set min/max attributes...

```suggestion
                // we use a type="text" input here, so we must set some a11y attributes
                // which we would otherwise get for free with a type="number" input
                aria-valuemin={0}
                aria-valuenow={valueNumber}
                aria-valuemax={getTimeUnitMax(unit)}
```